kinto/__main__.py

```diff
@@ -76,6 +75,7 @@ def main(args=None):
             try:
                 import psycopg2  # NOQA
             except ImportError:
+                import pip
```

Comment:
does this really speedup the startup time? If yes, do you know why? (importing pip shouldn't take that long!)